source/connectors/kafka-connect-migration.txt

```diff
@@ -0,0 +1,59 @@
+.. _kafka-connect-migration:
+
+==========================
+Migrate from Kafka Connect
+==========================
+
+.. default-domain:: mongodb
+
+This section lists the changes needed to migrate from the deprecated,
+community-developed version of the Kafka Connector (hosted in the
+`hpgrahsl/kafka-connect-mongodb GitHub repository
+<https://github.com/hpgrahsl/kafka-connect-mongodb>`_) to the `official
+connector <https://github.com/mongodb/mongo-kafka>`_.
+
+Once your project build file is updated to reference the official Kafka
+Connect library, use the following information to migrate your
+project.
+
+
+Package and Class Name Changes
+------------------------------
+
+- The package name for the Kafka Connect library used was changed from
+  ``at.grahsl.kafka.connect.mongodb`` to ``com.mongodb.kafka.connect``
+
+- The ``MongoDbSinkConnector`` class has been renamed ``MongoSinkConnector``
+
+- The ``MongoDbSinkConnectorConfig`` class has been split into
+  ``sink.MongoSinkConfig`` and ``sink.MongoSinkTopicConfig`` classes. Refer to
+  the source for a complete list of properties in `MongoSinkConfig
+  <https://github.com/mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/MongoSinkConfig.java>`_
+  and `MongoSinkTopicConfig
+  <https://github.com/mongodb/mongo-kafka/blob/master/src/main/java/com/mongodb/kafka/connect/sink/MongoSinkTopicConfig.java>`_.
+
+Configuration Changes
+---------------------
```

Comment:
This feels like a procedural step: **Update Your Kafka Configuration**

Reply:
It's not super clear what config these changes happen to. Is it a MongoDB config file or something in Kafka?

Reply:
Appears to be both source and sink connector configuration files.
I think the scope of this task changed from reformatting to rewriting the guides late last week. I'll probably need a resource to clarify a number of questions I have due to the current state of the documentation.